static/js/interactive-map/PinClusterer/PinClusterer.js

```diff
@@ -4,6 +4,33 @@ import { PinOptions } from '../Maps/MapPin.js';
 import { PinProperties } from '../Maps/PinProperties.js';
 import { Type, assertType, assertInstance } from '../Util/Assertions.js';
 
+/**
+ * Represents a cluster of {@link MapPin}s on a map.
+ * @member {MapPin} clusterPin
+ * @member {MapPin[]} pins
+ * @see {PinCluster}
+ */
+class PinCluster {
+  /**
+   * @param {MapPin} clusterPin
+   * @param {MapPin[]} pins
+   */
+  constructor(clusterPin, pins) {
+    this.clusterPin = clusterPin;
+    this.pins = [...pins];
+  }
+
+  /**
+   * Returns true if the cluster contains the pin with the given id, false otherwise
+   *
+   * @param {string} id The unique identifier for the pin
+   * @returns {boolean}
+   */
+  containsPin(id) {
+    return this.pins.filter((pin) => pin.getId() === id).length;
```

Comment:
nit: we can remove the parenthesis around `pin` since it's the only param to the function